__tests__/test.js

```diff
@@ -45,3 +105,79 @@ describe('getOrThrow', () => {
         expect(getOrThrow('TIMEZONE')).to.equal(process.env.TIMEZONE);
     });
 });
+
+describe('createIdObject', () => {
+    it('should create a object id', () => {
+        const returnObject = createIdObject(
+            ['mockedUser', 'mockedUser@user.com'],
+            mockedContributed
+        );
+
+        expect(returnObject.alternateId).to.equal('mockedUser@user.com');
+        expect(returnObject.github).to.equal('mockedUser');
+
+        const contributionsObject = [
```

Comment:
Would contributionObjects be a more accurate name, since it is an array of objects, each of which is a contribution?